interop-tests/src/tests/go_app/gen.go

```diff
@@ -0,0 +1,256 @@
+package main
+
+/*
+// Generated by rust2go. Please DO NOT edit this C part manually.
```

Comment:
Should the generated file be included in the version control?

Reply:
I'm neutral about it. It's not required but makes code readable for users who solely browse the repo. What's your recommendation?

Reply:
Pros:
- any change to it must be reviewed,
- somehow searchable.

Cons:
- will the changes to this file be actually reviewed or just blindly approved, especially when it'll be some obscure Go with C code?
- it introduces some noise, e.g., simple `Cargo.toml` or lockfiles are easily approved,
- while it allows some searching, it will also make general grepping more difficult - in my workflow, I'd need to disable looking in `.go` files to avoid noise.

All in all, if it's trivial to generate, I'd generate it on the fly and not add it to the version control.

Reply:
Removed